### DISCUSSION TITLE
Connect multiples servers in one dashboard

### DISCUSSION THREAD

**Discussion by Labs22** [telekom-security/tpotce#874] [Q&A]
Hey good evening everyone.
Im building a network of honeypots and I want to know if is possible to set multiples servers on cloud like aws on the same dashboard?

Any help I will be super glad.

Best regards

**Comment by t3chn0m4g3** [comment]
#437, #250

**Reply by Labs22** [reply]
> #437, #250

Hey man first thank you for the info I appreciate that but for now I will not able to build the network because Im in a litte problem.
I have make a OpenVPN server on aws, connected on the vpn on windows 10 and installed the framework honeypot on debian inside of the VirtualBox 500 gb 4 cores 8 threads and 11gb.
My question is , its possible to only open the ports on vpn and using the connection to start all services on the virtual machine?

I have to dothis for two reason one I cant open port on my pc and second my pc is powerful no need to take on cloud. So whats my best option ? Is there any guide to accomplish this?

Best regards.

**Reply by t3chn0m4g3** [reply]
No, there is no guide.
